libs/types/src/votingMethod.ts

```diff
@@ -1,5 +1,8 @@
-import { throwIllegalValue } from '@votingworks/utils'
-import { VotingMethod } from '@votingworks/types'
+import { VotingMethod } from './tallies'
+
+function throwIllegalValue(s: never): never {
```

Comment:
This is duplicated here to prevent a cycle between `utils` and `types`, right? Would be nice to explain as much in a comment.